js/components/Chart.vue

```diff
@@ -131,35 +130,6 @@ export default {
       this.chartData = data;
       this.filterEvents = filterEvents;
       this.warnings = warnings;
```

Comment:
I refactored all of the derived data into computed properties. This made recomputing the chart a simple matter of replacing the source data and calling `_makeChart`. Pretty nice.